### DISCUSSION TITLE
4.2.5 release notes don't cite Ruby 3.2.3 as requirement

### DISCUSSION THREAD

**Discussion by JetForMe** [mastodon/mastodon#29053] [General]
It says 3.0 - 3.2, but 4.2.4 release notes says "The recommended Ruby version has changed from 3.2.2 to 3.2.3" and sidekiq won't run without it.

**Comment by ssarhangi** [comment]
I have the same issue. My update bombed. Thanks

**Comment by JetForMe** [comment]
Yeah, I'm still trying to recover from not having 3.2.3.

**Reply by ssarhangi** [reply]
I installed ruby 3.2.3, then did the steps from https://docs.joinmastodon.org/admin/upgrading/ and I am up and running. You might have to do a systemctl start mastodon-web if it is not running. Altogether a very clumsy release update notes. Good Luck

**Reply by JetForMe** [reply]
I tried installing ruby using steps I found in my history, but that didn't work. Eventually I got that updated via rbenv, and rebuilt the release. BUT the release notes didn't include the asset precompile step. so now I gotta go do that.

I really wish the release notes were more explicit about each step.

**Reply by ssarhangi** [reply]
yes. Indeed. I updated ruby through rbenv, too. And then updated gems by running bundle install and yarn install --frozen-lockfile. And then restarted the services. Release notes were incomplete to say the least. Well good luck until the next upgrade 😀😀😀
